sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -318,6 +318,39 @@ private static int ParseHealthcareEntityIndex(string reference)
 
         #endregion
 
+        #region Extract Summary
```

Comment:
Adding lots of transforms in this class to convert from the internal models used by CodeGen to the public models exposed to users.